licensing.md

```diff
@@ -0,0 +1,27 @@
+# Licensing
+
+## Each repository should include a licence file
+
+This should be called `LICENCE` or `LICENCE.md`. "License" is the U.S. English spelling.
+
+GitHub.com will still show licence details for the British English spelling.
+
+## Use MIT
+
+At GDS we use the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
Fair enough, I don't actually have a real example so happy to leave it for when it becomes an issue